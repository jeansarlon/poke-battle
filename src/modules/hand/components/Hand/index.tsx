import React from 'react';
import { Card } from '..';

const Hand = () => (
  <div className="hand">
    hand
    <Card />
  </div>
);
export default Hand;
