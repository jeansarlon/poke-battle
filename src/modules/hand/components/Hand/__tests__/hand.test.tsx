import React from 'react';
import { render } from '@testing-library/react';
import Hand from '..';

describe('Hand', () => {
  test('should mount Component', () => {
    const { container } = render(<Hand />);

    const element = container.querySelector('.hand');
    expect(element).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});

