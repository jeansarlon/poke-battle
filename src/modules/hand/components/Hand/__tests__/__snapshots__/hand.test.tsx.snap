// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hand should mount Component 1`] = `
<div>
  <div
    class="hand"
  >
    hand
    <div
      class="card"
    >
      cards
    </div>
  </div>
</div>
`;
